masonry/src/widgets/image.rs

```diff
@@ -37,29 +39,12 @@ impl Image {
     /// By default, the Image will scale to fit its box constraints ([`ObjectFit::Fill`]).
     #[inline]
     pub fn new(image_data: ImageBuf) -> Self {
-        Self {
-            image_data,
-            object_fit: ObjectFit::default(),
-        }
-    }
-
-    /// Builder-style method for specifying the object fit.
-    #[inline]
-    pub fn fit_mode(mut self, mode: ObjectFit) -> Self {
```

Comment:
I think we need to have documentation with widgets about what properties can be used to customize them.